nat.tf

```diff
@@ -64,15 +64,18 @@ resource "aws_eip" "nat" {
     var.tags
   )
 
-  depends_on = [aws_internet_gateway.internet_gateway]
+  depends_on = [
+    aws_internet_gateway.internet_gateway,
+    var.module_depends_on
```

Comment:
```suggestion
    var.module_depends_on,
```

Reply:
Why is this important?

Reply:
If this is the enforced style then we should add it to the linters ( `terraform fmt` and `tflint` aren't complaining about it )

Reply:
> lgtm but would love to have a comma in the end when doing multi-line lists...

Can we add this to `terraform fmt` or `tflint` somehow?

Reply:
@mariux saw this?

Reply:
resolved in fd7f9a2